common/Geometry2d/Util.hpp

```diff
@@ -14,19 +14,23 @@
 #define M_E 2.718281828459045
 #endif
 
-constexpr float DegreesToRadians(float x) { return (x)*M_PI / 180.f; }
+constexpr float DegreesToRadians(float x) {
+    return (x) * static_cast<float>(M_PI) / 180.f;
+}
 
-constexpr float RadiansToDegrees(float x) { return (x)*180.f / M_PI; }
+constexpr float RadiansToDegrees(float x) {
+    return (x)*180.f / static_cast<float>(M_PI);
+}
 
 template <typename T>
 static inline T sign(T f) {
     if (f < T(0)) {
         return -1;
-    } else if (f > T(0)) {
+    }
+    if (f > T(0)) {
         return 1;
-    } else {
-        return 0;
     }
+    return 0;
```

Comment:
Is this something it complained about or is this more a style thing?

Reply:
It's a readability / style thing it complained about: `readability-else-after-return` [link](https://clang.llvm.org/extra/clang-tidy/checks/readability-else-after-return.html)